tests/conftest.py

```diff
@@ -0,0 +1,106 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Common testing tools for OAuth2Client tests.
+"""
+
+import sys
+import tempfile
+
+import pytest
+from six.moves import reload_module
+
+from oauth2client import util
+
+
+def setup_sdk_imports():
+    """Sets up appengine SDK third-party imports."""
+    if 'google' in sys.modules:
+        # Some packages, such as protobuf, clobber the google
+        # namespace package. This prevents that.
+        reload_module(sys.modules['google'])
+
+    # This sets up google-provided libraries.
+    import dev_appserver
+    dev_appserver.fix_sys_path()
+
+    # Fixes timezone and other os-level items.
+    import google.appengine.tools.os_compat
+    (google.appengine.tools.os_compat)
```

Comment:
Then is there a comment-level indication that can be used instead of a needless expression? `# pylint: disable=unused-import` or something perfectly clear like that?